spec/javascripts/components/show-password-spec.js

```diff
@@ -14,7 +14,7 @@ describe('A show password component', function () {
           '</div>' +
         '</div>'
       )
-      new GOVUK.Modules.ShowPassword().start(element)
+      new GOVUK.Modules.ShowPassword(element[0]).init()
```

Comment:
I'd not normally push to remove jQuery from the tests as well, but you've done it in other places and it would feel cleaner if module was being passed `element` instead of `element[0]`. Non-blocking.

Reply:
I can have a go, but given the amount of jQuery used in tests I'd rather give it another go at removing jQuery from tests separately, not to stall this PR.

Reply:
Fair point, non-blocking.